docker-compose.yml

```diff
@@ -1,32 +1,35 @@
-version: '2'
+version: '3'
+
 services:
-    image: sabaitech/seafile:latest
-    container_name: seafile
+  seafile:
+    build: .
     ports:
      - "8000:8000"
      - "8082:8082"
+    # connect localhost ports for oauth2 callback
+    network_mode: "host"
```

Comment:
I am not sur it is a good idea to use the 'host' mode as, if I am not mistaken, it removes the network isolation.